packages/expo-modules-core/ios/ModuleRegistryAdapter/EXModuleRegistryAdapter.m

```diff
@@ -70,6 +73,7 @@ - (instancetype)initWithModuleRegistryProvider:(EXModuleRegistryProvider *)modul
   // subclass EXViewManagerAdapter, so RN expects to find EXViewManagerAdapter
   // exported.
   [extraModules addObject:[[EXViewManagerAdapter alloc] init]];
+  [extraModules addObject:[[ViewModuleWrapper alloc] initWithDummy:nil]];
```

Comment:
`ViewModuleWrapper` is a base class of our swifty view managers, but `init` with no arguments would throw fatal exception, so I omit this by another convenience init.